src/common/darktable.c

```diff
@@ -146,6 +146,11 @@ static int usage(const char *argv0)
 #endif
   printf("  --dump-pfm <module>\n");
   printf("  --dump-pipe <module>\n");
+  printf("  --bench-module <module>\n");
```

Comment:
Probably a merge conflict resolution that reintroduce code removed here f4356b88086fb483275d1631b2443f43222e632f. This whole section can be removed as the actual Windows specific help line has been moved at the end of the options.